scripts/nf-core

```diff
@@ -39,30 +39,32 @@ def nf_core_cli(verbose):
     help = "Execute additional checks for release-ready workflows."
 )
 def lint(pipeline_dir, release):
-    """Check pipeline against nf-core guidelines"""
+    """ Check pipeline against nf-core guidelines """
 
-    # Create the lint object
-    lint_obj = nf_core.lint.PipelineLint(pipeline_dir)
+    # Run the lint tests!
+    nf_core.lint.run_linting(pipeline_dir, release)
 
-    # Run the linting tests
-    try:
-        lint_obj.lint_pipeline(release)
-    except AssertionError as e:
-        logging.critical("Critical error: {}".format(e))
-        logging.info("Stopping tests...")
-        sys.exit(1)
-
-    # Print the results
-    lint_obj.print_results()
+@nf_core_cli.command()
+@click.argument(
+    'pipeline_dir',
+    type = click.Path(exists=True),
+    required = True,
+    metavar = "<pipeline directory>"
+)
+@click.option(
+    '--version',
```

Comment:
I think this flag is confusing, as this usually gives you the version number of the command-line tool. And I think we should stick to this practice.

What about `--set-version` or `--make-version`?

Reply:
Ah, good spot! Yes, or even just a positional argument now that it's mandatory anyway..

```
nf-core release . 1.4
```

Reply:
also fine :)

Reply:
Updated 👍